general_rules/Robots-OPL.tex

```diff
@@ -47,4 +50,11 @@ \subsubsection{Audio output plug}\label{rule:roobt_audio_out}
 		\item It is the responsibility of the teams to plug in the transmitter before a test, to check the sound system, and to hand over the transmitter to next team.
 		\item Do not rely on the sound system! For fail-safe operation and interacting with operators make sure that the sound system is not needed, e.g., by having additional speakers directly on the robot.
 \end{compactitem}
-\end{enumerate}
\ No newline at end of file
+\end{enumerate}
+
+
+
+
+\subsubsection{Appearance}
+\label{rule:robots_appearance}
```

Comment:
I remember it wasn't because several teams in 2010~2012 tried to stick OpenCV/AR-markers on the manipulator's terminal effector for closed loop control while grasping and TC required to remove the stickers to pass RIPS (my former team was one of the affected ones in 2010, so we decided to have a blue hand, and a pink hand). And those were pre-kinect times.

Again, I am not against allowing markers on the robot (although in my opinion it looks pretty nasty), I am just trying to promote consistency.